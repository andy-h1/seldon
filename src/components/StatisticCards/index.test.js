import React from 'react';
import { render } from '@testing-library/react';
import { StatisticCards } from '.';

describe('Cards Statistic component', () => {
  it('should render', () => {
    const { container } = render(<StatisticCards />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
