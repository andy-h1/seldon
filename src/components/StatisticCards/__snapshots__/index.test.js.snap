// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards Statistic component should render 1`] = `
<div
  class="sc-hKgILt kvtJcu"
>
  <div
    class="sc-bdfBwQ ezRtVh"
  >
    <p
      class="sc-gsTCUz jPeWXq"
    >
      Played
    </p>
    <p
      class="sc-dlfnbm bfXIvc"
    >
      1254
    </p>
  </div>
  <div
    class="sc-bdfBwQ ezRtVh"
  >
    <p
      class="sc-gsTCUz jPeWXq"
    >
      Finished
    </p>
    <p
      class="sc-dlfnbm bfXIvc"
    >
      809
    </p>
  </div>
  <div
    class="sc-bdfBwQ ezRtVh"
  >
    <p
      class="sc-gsTCUz jPeWXq"
    >
      Likes
    </p>
    <p
      class="sc-dlfnbm bfXIvc"
    >
      114
    </p>
  </div>
  <div
    class="sc-bdfBwQ ezRtVh"
  >
    <p
      class="sc-gsTCUz jPeWXq"
    >
      Comments
    </p>
    <p
      class="sc-dlfnbm bfXIvc"
    >
      27
    </p>
  </div>
</div>
`;
